solutions/cses-1081.mdx

```diff
@@ -145,3 +147,24 @@ int main() {
 	}
 }
 ```
+</CPPSection>
+<PySection>
+```py
+MAXX = int(1e6 + 5)
```

Comment:
(optional)
`MAX_N` instead?

Reply:
i originally had it as maxn it was too confusing because there is an actual N in the problem